modules/exploits/windows/local/appxsvc_hard_link_privesc.rb

```diff
@@ -122,7 +125,9 @@ def cleanup
     folder_path = expand_path("%TEMP%\\etw")
     dir_rm(folder_path)
 
-    write_file(@rtf_path, @original_data)
+    unless @rtf_path.nil?
+      write_file(@rtf_path, @original_data)
+    end
```

Comment:
Meterpreter crashes if these are `nil`, which it is if the exploit didn't even start - decided to fix it while I was here.